.github/workflows/test-release.yml

```diff
@@ -0,0 +1,130 @@
+name: 'Test Release Sedge PPA'
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'The version number (e.g: v0.1.2) of the release you want to push'
+        required: true
+        default: 'master'
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  sedge-darwin:
+    name: Build sedge darwin
+    runs-on: macos-latest
+    env:
+      VERSION: ${{ github.event.inputs.tag }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.18.3'
+
+    - run: chmod +x ./scripts/build-go-darwin-binaries.sh && ./scripts/build-go-darwin-binaries.sh
+
+    - uses: actions/upload-artifact@master
+      name: Uploading sedge darwin amd64 package
+      with:
+        name: sedge-${{env.VERSION}}-darwin-amd64
+        path: build/sedge-${{env.VERSION}}-darwin-amd64
+
+    - uses: actions/upload-artifact@master
+      name: Uploading sedge darwin arm64 package
+      with:
+        name: sedge-${{env.VERSION}}-darwin-arm64
+        path: build/sedge-${{env.VERSION}}-darwin-arm64
+
+  sedge-linux:
+    name: Build sedge linux
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.event.inputs.tag }}
+    needs: sedge-darwin
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.18.3'
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - run: chmod +x ./scripts/build-go-linux-binaries.sh && ./scripts/build-go-linux-binaries.sh
+
+    - uses: actions/upload-artifact@master
+      name: Uploading sedge linux amd64 package
+      with:
+        name: sedge-${{env.VERSION}}-linux-amd64
+        path: build/sedge-${{env.VERSION}}-linux-amd64
+
+    - uses: actions/upload-artifact@master
+      name: Uploading sedge linux arm64 package
+      with:
+        name: sedge-${{env.VERSION}}-linux-arm64
+        path: build/sedge-${{env.VERSION}}-linux-arm64
+
+  publish-ppa:
+    name: "Publishing Sedge to PPA repository"
+    runs-on: ubuntu-latest
+    env: 
+      VERSION: ${{ github.event.inputs.tag }}
+      PPA_GPG_KEYID: ${{ secrets.PPA_GPG_KEYID }}
+    needs: publish-github
```

Comment:
You need to change this as publish-github is no longer in test build

Reply:
Fixed and rechecked the code. Please verify. Thanks